### PR TITLE
Perceiver Classifier for ADNI Alzheimer's dataset

### PR DESCRIPTION
**Pull request by zo1nk3dd**
I implemented the Perceiever model in pytorch to classify Alzheimer's disease from brain scans. Whilst I successfully got the model working and training, it did not really improve it's performance after a few hours of training. I ended up with around 50% accuracy on the test set.

### CODE REVIEW

**Comment by shakes76** [comment]
## Good Practice (Design/Commenting, TF/Torch Usage)
Adequate use and implementation (convergence issues) -2
Good spacing and comments
Header blocks missing -1
## Recognition Problem 
Solves problem
Driver Script present
File structure present
Shows Usage & Demo & Visualisation & Data usage 
Module present
Commenting
No Data leakage (validation set) -1
Difficulty: Hard
## Commit Log
Meaningful commit messages, could be more informative -1
Progressive commits used
## Documentation
ReadMe acceptable/good
Good Description and Comments 
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
No Feedback required
Request Description OK, could be more descriptive -1

Finalised now

**Comment by LinfengLiu98** [comment]
Model design: Good
Data split: did training and test split but missing a validation set during training
Training process: The training loss is not decreasing and training accuracy is not improving. Reasons were mentioned in the README.
Commit logs: Good, could have more detailed descriptions
README: Need to put in hyperparameters.

Overall, the student tried to solve the problem using a Perceiver but the training loss and the training ACC is not shown properly.
